index.js

```diff
@@ -96,8 +105,26 @@ const ScrollableTabView = React.createClass({
     }
   },
 
+  updateSceneKeys(children, sceneKeys = [], currentPage) {
```

Comment:
Is this correct if ```sceneKeys``` would be a Set?
I'm looking to log and this function add a key of ```currentPage``` to ```sceneKeys```?
What is the difference?